src/supermarket/app/controllers/api/v1/cookbook_versions_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Api::V1::CookbookVersionsController < Api::V1Controller
-  before_action :check_cookbook_name_present, only: [:foodcritic_evaluation, :collaborators_evaluation]
-  before_action :check_authorization, only: [:foodcritic_evaluation, :collaborators_evaluation]
-  before_action :find_cookbook_version, only: [:foodcritic_evaluation, :collaborators_evaluation]
+  before_action :check_cookbook_name_present, only: [:foodcritic_evaluation, :collaborators_evaluation, :publish_evaluation]
+  before_action :check_authorization, only: [:foodcritic_evaluation, :collaborators_evaluation, :publish_evaluation]
+  before_action :find_cookbook_version, only: [:foodcritic_evaluation, :collaborators_evaluation, :publish_evaluation]
```

Comment:
Do you have any thoughts on how we can give this list a name and manage its members in one place?